10up/simple-podcasting
# Feature: Cancel edit mode

### Description of the Change

Adds the ability for the user to edit their podcasting selection without going into the current "editor" mode and losing their currently selected podcast.

### Alternate Designs

Using a custom toggle with the current editor mode and a "cancel" button.

### Benefits

- Allows the user to edit their media selection without losing their current selection if they decide to cancel
- Aligns with the same editing pattern as the core audio block for consistency with WP core

### Possible Drawbacks

May be initially confusing if a user/editor is used to the current paradigm of editing.

### Verification Process

1. Create a new podcast block
2. Add a media file
3. Click the "Replace" dropdown
4. Click "Media Library" to replace file, abort
5. Confirm the media wasn't replaced
6. Click "Media Library", replace file
7. Confirm the file was replaced

### Checklist:






### Applicable Issues

#62 

### Changelog Entry

#### Removed
- Editing mode
#### Added
- Core `MediaReplaceFlow` to edit the podcast media
